src/constants/index.ts

```diff
@@ -0,0 +1,19 @@
+import { range } from 'lodash';
+import { getYear } from '../utils/date';
+
+export const MONTH = [
```

Comment:
Consider creating a "constants" folder under "src/core" where all the main reusable non-react things will be stored.

In this particular case all you have to do is to move your "src/contants" under "src/core".

Reply:
Thank you. Can I store the folder "types" there?

Reply:
Sure thing, "src/core" is used for any core level reusable functionality.
Examples of it might be API, I18Y, PropTypes, Types/Intefaces, constants, etc.